src/main/java/uk/gov/hmcts/reform/sscs/config/HighLevelDataSetupApp.java

```diff
@@ -70,14 +70,21 @@ public void addCcdRoles() {
 
     @Override
     protected List<String> getAllDefinitionFilesToLoadAt(String definitionsPath) {
-        if (environment.name().toUpperCase().equals(CcdEnvironment.PREVIEW.name())) {
-            return List.of(
-                    String.format("definitions/bulkscan/CCD_BulkScanningDefinition_%s.xlsx", "PR"),
-                    String.format("definitions/benefit/CCD_SSCSDefinition_%s.xlsx", "PR")
-            );
+        boolean isPreview = environment.name().equalsIgnoreCase(CcdEnvironment.PREVIEW.name());
+        boolean isHearingsPr = "yes".equalsIgnoreCase(System.getenv("HEARINGS_ENABLED"));
+
+        if (isPreview) {
+            if (!isHearingsPr) {
```

Comment:
Avoiding uploading bulkscan definition to AAT in hearings journey as its one more extra thing to delete when we close the PR.